ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/pages/legalRequirement.js

```diff
@@ -0,0 +1,14 @@
+class legalRequirements {
+    selectProject() {
```

Comment:
Is this function selecting a project?
It looks like its doing something else/

Reply:
It's a function, something I missed?